crates/dojo/core-cairo-test/src/tests/model/model.cairo

```diff
@@ -26,6 +26,74 @@ struct Foo2 {
     v2: u32
 }
 
+#[derive(Copy, Drop, Serde, Debug, IntrospectPacked)]
+#[dojo::model(version: 2)]
```

Comment:
Just wondering if we need this version number... Because before it was to support a breaking change. But actually, if the class hash changes, we know something has to be upgraded.

Also, imagine that the model contrat we generate changes.. but nothing breaking. Only more data that we didn't have before that we are adding. Then a user will not be able to actually upgrade.

So we may remove the version concept, and accept an upgrade if the class hash is different, even if the type is actually the same (same members, same types).

Reply:
Yes, you're right ! 👍